console/src/app/pages/users/user-detail/user-detail.module.ts

```diff
@@ -88,7 +88,7 @@ import { MetadataModule } from 'src/app/modules/metadata/metadata.module';
     DetailFormMachineModule,
     WarnDialogModule,
     MatDialogModule,
-    QrCodeModule,
+    QRCodeModule,
```

Comment:
does this need changing?

Reply:
yes.. the old library did not yet support angular 15 as the peer dependencies were conflicting 👍

Reply:
Did test it 👍

Reply:
ok